__tests__/app.test.js

```diff
@@ -122,3 +118,37 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe('GET /api/articles/:article_id/comments', () => {
+    it('200 - should respond with an array of comments with the correct keys', () => {
+        return request(app)
+            .get('/api/articles/1/comments')
+            .expect(200)
+            .then(({ body }) => {
+                expect(Array.isArray(body.comments)).toBe(true)
```

Comment:
can we refactor this test to test every single comment in the array for those properties. for article_id test the exact article_id as you know it should be 1